src/components/WatchLater.jsx

```diff
@@ -0,0 +1,42 @@
+import { useSelector, useDispatch } from 'react-redux'
+import { Link } from 'react-router-dom'
+import watchLaterSlice from '../data/watchLaterSlice'
+import Movie from './Movie'
+import '../styles/starred.scss'
+
+const WatchLater = ({viewTrailer}) => {
```

Comment:
code of this component repeats component `Starred `. Please reuse the code instead of duplicating it